autom4te.cache/requests

```diff
@@ -0,0 +1,252 @@
+# This file was generated.
```

Comment:
If autogen.sh is added, we will also need a .gitignore file, which will ignore the files in autom4te.cache and others, notably Makefile.ins if we're going to full autogen.sh mode.  Otherwise I'm not sure we should check in the Makefile.in changes for a newer version of automake which will revert to the older version when regenerated on jelly.

Reply:
I forgot to remove that!
OR
should we add it?

Reply:
We decided that the 1.8 branch would not get the autotools changes that went to what became 1.10. autogen.sh and the updated configure options should not be propagated to 1.8 without more discussion.